File: src/com/company/statically.js
class Statically {

   glue = "Epoxy";
jello = "Jello";

    moveAndShake() {
        console.log("Never stagnate in life.  Be agile, change, and improve...");
    }
static explain() {
        stubborn();
        console.log(this.glue);
        console.log('static is the same as adding it to the class and in java it is different. I dont really understand')
    }

}
 function stubborn() {
        stats.moveAndShake();
        console.log("Please don't make me change... I just don't like it...");
    }

const stats = new Statically();

stats.moveAndShake();
stubborn();
console.log(Statically.explain());

console.log(stats.glue);
console.log(stats.jello);

//Once you have finished getting statistically.js to run, refactor the code here in Java.

File: src/com/company/Statically.java
package com.company;

public class Statically {
    public static void main(String[] args){
        String glue = "Epoxy";
        String jello = "Jello";
        moveAndShake();
        stubborn();
        explain(glue);
        System.out.println(jello);
        System.out.println(glue);
    }
    public static void moveAndShake(){
        System.out.println("Never Stagnate in life. Be agile, change, and improve....");
    }
    public static void stubborn(){
        moveAndShake();
        System.out.println("Please dont make me change! I hate change");
    }
    public static void explain(String glue){
        stubborn();
        System.out.println(glue);
    }
}
